io/src/main/scala/sbt/io/Path.scala

```diff
@@ -375,7 +375,7 @@ sealed abstract class PathFinder {
    * The files returned by this method will reflect the underlying filesystem at the time of calling.
    * If the filesystem changes, two calls to this method might be different.
    */
-  final def get: Seq[File] = {
+  final def get(): Seq[File] = {
```

Comment:
Are we adding empty parentheses because it forces evaluation?

Reply:
Would this logic apply to `getURLs`?

Reply:
Yes to both. I'll look at the uses of get and see if there are more to add.